shared/deps/serial/serial_host_send_receive.py

```diff
@@ -191,6 +191,7 @@ def main():
 
     vport = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
     vport.connect(sock)
+    vport.setsockopt(1, socket.SO_SNDBUF, p_size)
```

Comment:
I haven't seen the possible reason to have to do so, so would please explain more on it? For my point of view, `shutdown()`/`close()` would sufficiently resolve the issue.

Reply:
@luckyh To be honest I don't really understand how it affects. I had same think with you, I suppose `close` should resolve the issue, but from the test result, only set send buffer size can resolve the issue.
With this set size line, no matter have shutdown and close lines or not, the test can pass.
Well without this line, no matter have shutdown and close lines or not, the test would fail.

Reply:
@vivianQizhu well, that might be a product bug or something else, so better to figure out it.